app/views/layouts/design_system.html.erb

```diff
@@ -18,6 +18,19 @@
 
   <div class="govuk-width-container">
     <main class="govuk-main-wrapper" id="main-content" role="main">
+      <% [:success, :info, :warning, :danger, :notice, :alert].select { |k| flash[k].present? }.each do |k| %>
```

Comment:
Should we be using the design system components ([Error Alert](https://components.publishing.service.gov.uk/component-guide/error_alert), [Success Alert](https://components.publishing.service.gov.uk/component-guide/success_alert), [Notice](https://components.publishing.service.gov.uk/component-guide/notice)), rather than copying this pattern from the legacy layout template?

Reply:
I thought that implementing the design system component would be a separate task in of itself. I think the flash message probably shouldn't have been removed initially, which is why I'm putting it back for now.

Reply:
Maybe we need a team discussion around this, as I would expect that without using the components from the design system, the look of the flash messages won't be acceptable for releasing these pages, in which case we need to get a story for doing that done ASAP.

I think we probably need a discussion around how we're going to release pages (i.e. turn the default state of the toggle to on) anyway.

Reply:
I have implemented the [flash messages as part of the actual page content itself](https://github.com/alphagov/publisher/pull/2031).